apps/prairielearn/src/pages/instructorAssessmentAccessOverrides/instructorAssessmentAccessOverrides.ejs

```diff
@@ -0,0 +1,321 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <%- include('../partials/head'); %>
+</head>
+<body>
+  <script>
+    function toDatetimeLocal(date) {
```

Comment:
This does not handle seconds; it should.

Reply:
done in [6e6ae7b](https://github.com/PrairieLearn/PrairieLearn/pull/9103/commits/6e6ae7bdfb095e3be7bd941b0a23e25bbc3af309)

Reply:
^ that being said function deleted later